app/controllers/patients_controller.rb

```diff
@@ -56,6 +56,17 @@ def data_entry_create # temporary
       render 'data_entry'
     end
   end
+
+
+  def fetch_clinic_naf
```

Comment:
What are your thoughts about moving this to the `clinics` controller? Can rig it up something like `ClinicsController#naf_status`?